java/pom.xml

```diff
@@ -35,6 +35,7 @@
     <dep.guava.version>30.1.1-jre</dep.guava.version>
     <dep.netty.version>4.1.72.Final</dep.netty.version>
     <dep.jackson.version>2.11.4</dep.jackson.version>
+    <dep.jackson-bom.version>2.13.2.20220328</dep.jackson-bom.version>
```

Comment:
Why the discrepancy, why can't we just edit dep.jackson.version above?

Reply:
I just updated `<dep.jackson.version>2.13.2</dep.jackson.version>` but consider if we maintain that with only this parameter version all libraries are going be updated to 2.13.2, but, for jackson-databind we need 2.13.2.2 and this is handled by [jackson BOM](https://github.com/FasterXML/jackson-bom/blob/jackson-bom-2.13.2.20220328/pom.xml#L62) `<dep.jackson-bom.version>2.13.2.20220328</dep.jackson-bom.version>`

Reply:
I don't think we want to version the Jackson libraries separately though. If we're upgrading one we should upgrade all.

Reply:
At summary we need two variables:
- For jackson-databind: cve 2.13.2.2 (We have two option: modify all pom.xml and set this version OR delegate this to jackson-bom)
- For jackson-core, jackson-annotation, jackson-dataformat-yaml, jackson-datatype-jsr310: 2.13.2

Reply:
Ah, they only published revisions for one library…Alright then. Can we make it `dep.jackson.databind.version` or something and note that upstream versioning is out of sync? It's unclear what `bom` is supposed to control.

Reply:
Oh, sorry, we depend on `bom` indeed. Alright then.